Header header

float32 Lat_m
float32 Long_m

float32 Lat_deg
float32 Long_deg
